docker/build-tools/Dockerfile

```diff
@@ -359,9 +359,10 @@ ENV GO111MODULE=on
 ENV GOPROXY=https://proxy.golang.org
 ENV GOSUMDB=sum.golang.org
 ENV GOROOT=/usr/local/go
-ENV GOPATH=/home/go
-ENV GOCACHE=/home/gocache
-ENV PATH=/usr/local/go/bin:/home/go/bin:/usr/local/google-cloud-sdk/bin:$PATH
+ENV GOPATH=/go
+ENV GOCACHE=/gocache
+ENV GOBIN=/tmp/out
```

Comment:
is this to be a hostmount?  /tmp is already hostmounted in its entirety by the cni repo and required for correct e2e functional tests.